templates/definition/meter/deye-hybrid-hp3.yaml

```
@@ -56,10 +56,8 @@ render: |
       address: 522 # "Total_GridBuy_Power Wh"
       type: holding
       decode: uint32s
-    {{- if eq .firmware1098 "true" }}
-    scale: 1   # Firmware >= 1098: scaling factor is 1 kWh
-    {{- else }}
-    scale: 0.1 # Older firmware: scaling factor is 0.1 kWh
+    {{- if eq .firmware1098 "false" }}
```

```suggestion
    {{- if ne .firmware1098 "true" }}
```

...denn auf alten Installationen ist das gar nicht gesetzt solange der Anwender es nicht wieder neu abspeichert. Das ist leider etwas tricky.